### PR TITLE
Add new JVM options to cassandra-env.sh

### PR DESCRIPTION
**Pull request by ko-be**
Add new JVM options to cassandra-env.sh, currently working well on Production.

### CODE REVIEW

**Comment by bossjones** [comment]
so just to confirm, you need to remember to set that otherwise it'll error out when cassandra tries to start ?

**Comment by bossjones** [comment]
scratch that, I was looking at `JVM_OPTS="$JVM_OPTS -XX:NewSize=${HEAP_NEWSIZE} -XX:MaxNewSize=${HEAP_NEWSIZE}` and just realized it's using the same var for both options, cool.

LGTM

File: metadata.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "maintainer_email": "michael@clojurewerkz.org",
   "license": "Apache 2.0",
   "description": "Installs/configures Apache Cassandra",
-  "version": "2.10.0",
+  "version": "2.10.1",
```
Comment (chrishenry) [inline metadata.json]: nice, I always forget the stupid `metadata.json` file

**Comment by chrishenry** [comment]
LGTM

**Pull request merged**